docs/source/multi_gpu.rst

```diff
@@ -51,6 +51,18 @@ This will make your code scale to any arbitrary number of GPUs or TPUs with Ligh
         z = z.type_as(x, device=self.device)
 
 The LightningModule knows what device it is on. You can access the reference via `self.device`.
+Sometimes it is necessary to store tensors as module attributes. However, if they are not parameters they will
+remain on the CPU even if the module gets moved to a new device. To prevent that and remain device agnostic,
+register the tensor as a buffer with :meth:`~torch.nn.Module.register_buffer`.
+
+.. testcode::
+
+    # in your Lightning- or nn.Module
+    def __init__(self):
```

Comment:
this is strange - stand-alone init without class definition?

Reply:
what do you mean? we also have standalone forward etc. see above

Reply:
Ok, I would at least add the class definition to see from what class it is derived...